### ISSUE TITLE
Lightdock reproducibility across different plataforms using the same seed

### ISSUE THREAD

**Issue by alexandrefassio** [lightdock/lightdock#82]
Hi there,

I ran Lightdock on a Mac and a Linux computer using the same inputs and parameters, including seeds. After docking, I filtered out complexes by score. As expected, the setup files, lightdock.info, and files in init/ are all identical.

However, when I compared the ranking files from the different platforms, I observed the results were different. Curiously, though, when I opened the top structures in Pymol from each run in a pair-wise way, I observed the poses were very close. For example, the RMSD for the top 1 was 0.16. Note that both structures had different names and scores, as shown below. 

swarm_43_127.pdb   80.288  0.667                vs             swarm_43_0.pdb   78.505  0.667

I know that sometimes, when we run software across different platforms, we may get slightly different results despite using the same seeds. So, I just wanted to confirm that this is an expected behavior of Lightdock.

Thanks

**Comment by brianjimenez** [comment]
Hi @alexandrefassio, I am afraid that these numerical differences are inherent to the difference in architecture (CPU on Mac vs Linux), and only the pure Python scoring functions would deliver exact results on different platforms (Python numerical objects keep and deal with precission). In the case of `fastdfire`, the default scoring function, it is implemented in C and that for sure will jeopardize numerical stability.

In summary, this is expected behavior in LightDock.
